src/devices/video/huc6270.h

```diff
@@ -21,15 +286,24 @@ class huc6270_device : public device_t
 	void set_vram_size(uint32_t vram_size) { m_vram_size = vram_size; }
 	auto irq() { return m_irq_changed_cb.bind(); }
 
+	// 8 bit I/O
 	u8 read(offs_t offset);
 	void write(offs_t offset, u8 data);
+	// TODO: 16 bit I/O used?
+	/*
+	u16 read16(offs_t offset);
+	void write16(offs_t offset, u16 data);
+	*/
+	// VD0-7, SPBG output 
 	u16 next_pixel();
 	inline u16 time_until_next_event()
 	{
 		return m_horz_to_go * 8 + m_horz_steps;
 	}
 
+	// /VSYNC pin related
 	DECLARE_WRITE_LINE_MEMBER( vsync_changed );
+	// /HSYNC pin related
```

Comment:
Function signatures are clear enough to define those as pins, commentary is superfluous.